src/controlnet_aux/tests/test_processor.py

```diff
@@ -0,0 +1,89 @@
+"""Test the Processor class."""
```

Comment:
We overlapped here and I added initial unit tests in #49 but using pytest.  @patrickvonplaten - which approach would you prefer?

Reply:
I added a pytest script too

Reply:
Think we can def add two test files here no? Would be very nice to have some tests :-)

Reply:
Think we can def add two test files here no? Would be very nice to have some tests :-)